src/app/modules/shared/components/dropdown/dropdown.component.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../../../../styles/colors.scss';
+
+.btn--switch {
+    font-size: 1rem;
+    border: 1px solid #c5c5c5;
```

Comment:
Import **#c5c5c5;** this color from `colors.scss`, you can use `lighten($primary-text, 30%)`

Reply:
done